third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/fwtransport/framework_utils.go

```diff
@@ -19,7 +19,7 @@ import (
 const uaEnvVar = "TF_APPEND_USER_AGENT"
 
 func CompileUserAgentString(ctx context.Context, name, tfVersion, provVersion string) string {
-	ua := fmt.Sprintf("Terraform/%s (+https://www.terraform.io) Terraform-Plugin-SDK/%s %s/%s", tfVersion, "terraform-plugin-framework", name, provVersion)
+	ua := fmt.Sprintf("kcc/1.117.0 (+https://github.com/GoogleCloudPlatform/k8s-config-connector) kcc/controller-manager/%s", provVersion)
```

Comment:
Maybe `//go:embed` would be the easiest approach to the version problem?  I think we can `go:embed` just a string.

Reply:
Done